app/services/database_health_checker.rb

```diff
@@ -0,0 +1,26 @@
+module DatabaseHealthChecker
+  module_function
+
+  Summary = Struct.new(:healthy, :result) do
+    def as_json(*args)
+      to_h.as_json(*args)
+    end
+  end
+
+  # @return [Summary]
+  def check
+    Summary.new(true, simple_query)
+  rescue => err
+    Summary.new(false, err.message)
```

Comment:
I've never seen error messages that include things like that? The query is a simple `SELECT 1` and does not support custom queries, so I don't see how this failing would return any other details like that?